src/index.html

```diff
@@ -1,17 +1,156 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      onsubmit="onSubmit()">
+      <fieldset class="fieldset">
+        <legend> Personal information </legend>
+        <div>
+          <label for="Surname"> Surname: </label>
+          <input
+            type="text"
+            name="Surname"
+            id="Surname"
+            required minlength="2"
+            maxlength="32"
+            autocomplete="off">
```

Comment:
```suggestion
          <input
            type="text"
            name="Surname"
            id="Surname"
            required minlength="2"
            maxlength="32"
            autocomplete="off"
          >
```
Закрывающую скобку располагай на одной линии с открывающей